docs/ADR/009-messaging-bus.md

```diff
@@ -0,0 +1,60 @@
+# ADR-008-open-source-license
```

Comment:
It will be better if this file is in another PR

Reply:
ooh bad handling